src/translations/fr.json

```diff
@@ -780,19 +792,32 @@
     "ShuntCurrentNumberOfSections": "Nombre de sections utilisées",
     "ShuntMaximumNumberOfSections": "Nombre de sections disponibles",
     "CreateShuntCompensator": "Créer un moyen de compensation statique (Linéaire)",
+    "CreateStaticVarCompensator": "Créer un CSPR",
+    "StaticVarCompensatorConnectivityTab": "Connectivité",
+    "StaticVarCompensatorSetPointsAndLimitsTab": "Consignes & Limites",
+    "StaticVarCompensatorAutomatonTab": "Automate",
+    "StaticVarCompensatorAdditionalInfosTab": "Compléments",
     "ShuntCompensatorErrorMaximumLessThanOne": "Le nombre maximum de sections doit être supérieur à 0",
     "ShuntCompensatorErrorCurrentLessThanZero": "Le nombre de sections utilisées doit être supérieur ou égal à 0",
     "ShuntCompensatorErrorCurrentLessThanMaximum": "Le nombre de sections utilisées doit être inférieur ou égal au nombre maximum",
     "ShuntCompensatorErrorQAtNominalVoltageLessThanZero": "La valeur de Q à Unom doit être supérieure ou égale à 0",
+    "StaticVarCompensatorErrorSMaxAtNominalVoltageGreaterThanSMin": "La valeur de susceptance maximum doit être supérieur ou égale à susceptance minimum",
+    "StaticVarCompensatorErrorSMinAtNominalVoltageLessThanSMax": "La valeur de susceptance minimum doit être inférieur ou égale à susceptance maximum",
+    "StaticVarCompensatorErrorQMaxAtNominalVoltageGreaterThanQMin": "La valeur de Q à Unom maximum doit être supérieur ou égale à Q minimum",
+    "StaticVarCompensatorErrorQMinAtNominalVoltageLessThanQMax": "La valeur de Q à Unom minimum doit être inférieur ou égale à Q maximum",
     "ModifyShuntCompensator": "Modifier un moyen de compensation statique (Linéaire)",
     "MaximumSectionCountMustBeGreaterOrEqualToOne": "Le nombre de gradins doit être supérieur ou égal à 1",
     "SectionCountMustBeBetweenZeroAndMaximumSectionCount": "La valeur de la prise courante doit être comprise entre 0 et le nombre de gradins",
     "maximumSectionCount": "Nombre de gradins",
     "sectionCount": "Gradin courant",
     "SwitchedOnMaxQAtNominalV": "Q enclenchée à tension nominale",
     "SwitchedOnMaxSusceptance": "Susceptance enclenchée",
-    "maxQAtNominalV": "Q installée à tension nominale",
-    "maxSusceptance": "Susceptance installée",
+    "maxQAtNominalV": "Q max à tension nominale",
```

Comment:
This gonna change few translations. Is it sure that it is what we want ?

Reply:
done